fang/fang_examples/blocking/simple_worker/src/main.rs

```diff
@@ -1,5 +1,5 @@
 use diesel::r2d2;
-use dotenvy::dotenv;
+use dotenv::dotenv;
```

Comment:
why can't we use dotenvy?

Reply:
We can use `dotenvy` but it should be changed on `Cargo.toml`

I can change it if you would like

Reply:
[all fang examples using dotenvy](https://github.com/ayrat555/fang/pull/125/commits/14ff2fbd0df23b629eba5490348012f008c822b3)